x-pack/plugins/security_solution/public/cases/components/create/index.tsx

```diff
@@ -28,7 +28,6 @@ import {
 } from '../../../shared_imports';
 import { usePostCase } from '../../containers/use_post_case';
 import { schema, FormProps } from './schema';
-import { InsertTimelinePopover } from '../../../timelines/components/timeline/insert_timeline_popover';
```

Comment:
Shall we remove 
x-pack/plugins/security_solution/public/timelines/components/timeline/insert_timeline_popover/index.tsx
as it seems that no where is using it.